devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -100,6 +100,15 @@ def parse(String description) {
 		if (cmd) {
 			result = zwaveEvent(cmd)
 		}
+
+		if (!state.setAssociation)
+		{
+			// This change is to fix the issue introduced in 28.11 where association to group ID 1 was not set at inclusion.
+			// we are going to set the association to group id 1 every time we hear anything from this device
```

Comment:
The `every time` part isn't true since this will only be attempted once

Reply:
Force push an update. Can you take a look?